src/Expensify.js

```diff
@@ -124,6 +124,14 @@ class Expensify extends PureComponent {
         return lodashGet(this.props, 'session.authToken', null);
     }
 
+    hideSplash() {
+        BootSplash.hide({fade: true}).then(() => {
+            // To prevent the splash from shifting positions we set status bar translucent after splash is hidden.
+            // on IOS it has no effect.
+            StatusBar.setTranslucent(true);
```

Comment:
`src/components/CustomStatusBar/index.android.js` mentions `StatusBar.setTranslucent` is only supported in android - is that going to cause any problems here?

Reply:
no its will only show a warning in development build on IOS. in production it simply does return;
https://github.com/facebook/react-native/blob/d6cd2e6559ff8698833dc277810e2e7e80af760a/Libraries/Components/StatusBar/StatusBar.js#L336